sematic/resolvers/silent_resolver.py

```diff
@@ -34,7 +54,11 @@ def _run_inline(self, future: AbstractFuture) -> None:
                     ),
                 )
             ):
-                value = future.calculator.calculate(**future.resolved_kwargs)
+                try:
+                    with timeout(timeout_seconds):
```

Comment:
`None` is a valid value for `timeout_seconds` here, so `int` would fail. Note that the line you referenced says "convert to seconds if it's not None. otherwise leave as None"